cloud/azure/bin/saml-keystore-setup

```diff
@@ -0,0 +1,68 @@
+#! /usr/bin/env bash
+source bin/lib.sh
+
+# DOC: Generate PKCS12 keystore and upload to Azure Blob Storage. 
+# Make sure to run the key_vault_setup script before running this one.
+# Required flags are:
+# g - Azure resource group
+# v - Azure key vault name
+# s - Azure storage account name
+# l - Azure region
+
```

Comment:
issue (blocking): the `bin/help` script won't print these lines without the `DOC: ` prefix